chain/client/src/client_actor.rs

```diff
@@ -457,6 +457,14 @@ impl Handler<NetworkClientMessages> for ClientActor {
                 }
                 NetworkClientResponses::NoResponse
             }
+            NetworkClientMessages::PartialEncodedChunkForward(forward) => {
+                if let Ok(accepted_blocks) =
+                    self.client.process_partial_encoded_chunk_forward(forward)
```

Comment:
we should probably log the error if it is not `near_chunks::Error::UnknownChunk`

Reply:
Done in https://github.com/nearprotocol/nearcore/pull/2769/commits/1e284e0b3d0ac42a5af9d817ee6b9b10210ffff6